src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="head_a"
+        href="index.html"
+      >
```

Comment:
The class name 'head_a' does not represent the content of the element. Class names should represent the meaning of the content, not the styles or tag names. Consider renaming this class to something more descriptive, such as 'header-logo' or 'header-link'.